packages/components/app/styles/components/form/file-input.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM > FILE-INPUT
+//
+
+@use "../../mixins/button" as *;
+
+.hds-form-file-input {
+  margin: -4px 0 -4px -4px; // offset shadow padding + default 1px spacing
+  padding: 3px 0 3px 3px; // add padding so button shadow is visible
+  font-family: var(--token-typography-font-stack-text);
+  border: 1px solid transparent;
+  border-radius: calc($hds-button-border-radius + 1px);
+  // font-size: 1.875rem; // we may want to use this to tweak the size of the text that contains the name of the file?
```

Comment:
I think Safari has a tiny `font-size` by default. It would be great if we can set a certain size, but following the pattern in other components, using a typography class may be the preferred way of setting a `font-size` – hence you may actually need `get classNames() { }`

Reply:
I pushed updates using the hds classnames to replace these font-styles earlier.

Safari is problematic but I'll continue to experiment with improving styles for it when the more final visual designs are ready.

Reply:
Yeah, I know, it's tricky. If there's no way around it, I think we can accept this visual difference.

Reply:
I did as much as I could with Safari. It is respecting the font size but I haven't found a way to add spacing between the button and the status message text.